utils/docker/Dockerfile.leap.15

```diff
@@ -14,19 +14,29 @@ ARG CB0
 
 # Use our own repo cache
 ARG REPO_URL=""
-ARG REPO_LEAP=""
-RUN if [ -n "${REPO_LEAP}" ]; then				                               \
-      rm -f /etc/zypp/repos.d/*.repo &&					                       \
-      zypper --non-interactive ar ${REPO_URL}${REPO_LEAP}/                     \
-           daos-stack-group-repo &&                                            \
-      zypper --non-interactive mr --gpgcheck-allow-unsigned                    \
-           daos-stack-group-repo;                                              \
-    fi;                                                                        \
-    zypper addlock fuse fuse-libs fuse-devel
+ARG REPO_DAOS=""
+ARG REPO_DISTRO=""
 
-# Make sure repos are up to date
-RUN zypper --non-interactive --no-gpg-checks --gpg-auto-import-keys refresh
+# If a DISTRO repo is provided then use it.
+RUN if [ -n "${REPO_DISTRO}" ]; then				                           \
+      rm -f /etc/zypp/repos.d/*.repo &&					                       \
+      zypper --non-interactive addrepo --gpgcheck-allow-unsigned-repo          \
+            ${REPO_URL}${REPO_DISTRO}/  daos-stack-distro-group-repo &&        \
+      zypper --non-interactive modifyrepo --gpgcheck-allow-unsigned-repo       \
+             daos-stack-distro-group-repo &&                                   \
+      zypper --non-interactive clean --all;                                    \
+    fi
```

Comment:
Interesting the order in this one is as I am suggesting above for EL*.

Reply:
Yes, it just works for zypper.